scss/_utilities/_variables.scss

```diff
@@ -15,7 +15,7 @@ $desktop: $larger;
 $susy: (
   container: 1440px,
   columns: 16,
-  gutters: 24px/64.5px,
+  gutters: (24/64.5),
```

Comment:
![it s a trap](https://cloud.githubusercontent.com/assets/105849/5728783/31055666-9b3b-11e4-9db2-d4d0e8a32702.gif)